bcda/web/v2/api_test.go

```diff
@@ -0,0 +1,1534 @@
+package v2
```

Comment:
Looks like a lot of `DRY` here.  I wonder if we can have a single `api_test.go` which covers both `v1` tests and `v2` tests with common code?  Or are they different enough that separate explicit tests are warranted?  I worry about maintaining duplicate code here.